toolsrc/bmp2c.c

```diff
@@ -6,6 +6,14 @@
 
 #define XPARENT (0xf7)	/* palette index representing transparent in BMP */
 
```

Comment:
This change is completely unrelated!

I appreciate your changes, but please file them as a separate PR. Also, there is already a setup for packed structs in https://github.com/fabiangreffrath/woof/blob/f0c5d0733f005b97cf7e69055573ef49914df7e4/Source/doomtype.h#L91, so please use this instead.